src/view.c

```diff
@@ -1049,6 +1049,20 @@ view_toggle_always_on_bottom(struct view *view)
 	}
 }
 
+bool
+view_is_omnipresent(struct view *view)
+{
+	assert(view);
+	return view->omnipresent;
+}
+
+void
+view_toggle_omnipresent(struct view *view)
+{
+	assert(view);
+	view->omnipresent = !view_is_omnipresent(view);
```

Comment:
So this could then just be a `view->visible_on_all_workspaces = !view->visible_on_all_workspaces`.